simtools/model/camera.py

```diff
@@ -392,34 +400,30 @@ def _find_neighbors(x_pos, y_pos, radius):
 
         Returns
         -------
-        neighbors: numpy.array_like
-            Array of neighbor indices in a list for each e.g., pixel.
+        list of lists
+            Array of neighbor indices in a list for each pixel
         """
-        points = np.array([x_pos, y_pos]).T
-        indices = np.arange(len(x_pos))
-        kdtree = KDTree(points)
-        neighbors = [kdtree.query_ball_point(p, r=radius) for p in points]
+        tree = KDTree(np.column_stack([x_pos, y_pos]))
```

Comment:
As these lines have been changed, would it be possible to add unit test coverage? This would make the change much clearer.